port/zos390/omrsl.c

```diff
@@ -288,7 +290,7 @@ omrsl_lookup_name(struct OMRPortLibrary *portLibrary, uintptr_t descriptor, char
 	}
 
 	if (address == NULL) {
-		Trc_PRT_sl_lookup_name_Exit2(name, argSignature, handle, 1);
+		Trc_PRT_sl_lookup_name_Exit2(name, argSignature, handle, retcode);
 		return 1;
```

Comment:
Should this return retcode? It seems safe to do so.

Reply:
Good idea, and I agree.  Updated under: e15cc29